packages/pipeline/src/utils.ts

```diff
@@ -0,0 +1,17 @@
+/*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0 */
+
+import { DEFAULT_BRANCH_NAME } from "./pdk-pipeline";
+
+export class PipelineUtils {
```

Comment:
Nit: it might be neater to have these as static members of the `PDKPipeline` construct rather than separate them out as they're quite closely related :)